### PR TITLE
ISSUE-109438 - Document running Pega without Cassandra

### PR DESCRIPTION
**Pull request by baylp2022**
ISSUE-109438 - Document running Pega without Cassandra

### CODE REVIEW

File: charts/pega/README.md [hunk 1]
```diff
@@ -762,6 +762,21 @@ dds:
   username: "dnode_ext"
   password: "dnode_ext"
 ```
+### Deploying Pega without Cassandra
+You may configure Pega without any Cassandra datastore. When deploying in this mode set `cassandra.enabled` to `false` and comment out or delete the `dds` section.
```
Comment (taz-pega-work) [inline charts/pega/README.md:766]: how about match the statement to the heading>>
To configure a Pega Platform deployment without a Cassandra datastore (DDS), set `cassandra.enabled` to `false` and comment out or delete the `dds` section.

File: charts/pega/values.yaml [hunk 2]
```diff
@@ -328,6 +328,8 @@ cassandra:
 
 # DDS (external Cassandra) connection settings.
 # These settings should only be modified if you are using a custom Cassandra deployment.
+# To run Pega without Cassandra comment out or delete the following dds section and set
```
Comment (taz-pega-work) [inline charts/pega/values.yaml:331]: slight tweak>>
To deploy Pega Platform without using Cassandra, comment out or delete the `dds` section and set
the `cassandra.enabled` property above to `false`.

**Review by taz-pega-work — COMMENTED** [review]
Please consider slight adjustments to wording and punctuation

**Review by taz-pega-work — APPROVED** [review]
looks good. thanks for patiently updating the content to add some clarity to the option.

**Review by MadhuriArugula — APPROVED** [review]

**Review by kishorv10 — APPROVED** [review]

**Comment by taz-pega-work** [comment]
merging after seeing everything is in order

**Pull request merged**